constraints_py36.txt

```diff
@@ -1 +0,0 @@
-pywinpty==2.0.2
```

Comment:
Only exists for Windows 3.6 compatibility - Ben H and I had to patch the release because of this a few months back.